backend/src/domain/models/Game.js

```diff
@@ -0,0 +1,46 @@
+
+/**
+ * Main game controller for the application.
+ * Controls the timing of each round and state of the game
+ */
+export class Game {
+  /**
+     * @param {*} io, server io socket connection
+     * @param {*} session, session for game to be played
+     * @param {*} swipeDeck, list of restaurants
+     */
+  constructor(io, session, swipeDeck) {
+    this.io = io;
```

Comment:
Games.js already has io and session as fields, is it needed again here?

Reply:
Session is needed because each game has an individual sesison. But i guess 'io', can be passed through method when needed, because all games will have the same parent io.

Reply:
Is it the io object we are passing, or the socket object? If its io object, I think I have a export {io} at the end of index.js. So that it can be imported from everywhere else.

Reply:
@Tianrens what do you mean, by Games.js already having io and session as fields? I don't see where they are being set in Games.js

Reply:
oh wait your right nvm

Reply:
:)